content/docs/user-guide/assets/asset-processor/asset-cache-server.md

```diff
@@ -172,19 +172,21 @@ The asset caching system is configured using opt-in patterns. There are many typ
 
 This ACS block allows users to configure the types of source assets that should be cached in the remote folder. The block is placed in the JSON path `/AssetProcessor/Settings/Server` object. The title must start with the prefix `"ACS "` to designate the object as a configuration block. The next part is either `"glob"` or `"pattern"` followed by the correct text for a wild card pattern or a regular expression; these are used to tag source assets that need to be cached.
 
+The ``name`` field is used for both the title of the configuration block in the GUI tool, but it is also possible to match all an asset pattern by setting the ``name`` to a name of an asset builder. For example, a configuration block can set the name to "Atom Image Builder" so that all the processed images will be cached.
+
 {{< note >}}
 The block should only be set to `"glob"` or `"pattern"`, not both.
 {{< /note >}}
 
 The `"checkServer"` Boolean flag is used to enable the block. The default value for `"checkServer"` is false, so to enable the ACS block the Boolean flag needs to be set to true.
 
-An example JSON ACS block:
+An example this JSON ACS block will cache all product assets built by the "XmlBuilderWorker" builder:
```

Comment:
Talk about how the name and glob works, is it 'and' or 'or' when in an ACS with a name. ie what is the glob adding to the ACS block below. with `name` it will cache all the outputs from the XMLBuilderWorker, so what is the glob modifying?

Reply:
The glob also can be used to match the patterns if some types of the asset builders want to be cached, but not the others. For example. if the team sets up "FBX ONLY" rule that caches only "*.fbx" then the rule's name could be "FBX ONLY" but with the *.fbx source files. If the "Scene Builder" was the name then all the source scene files would be cached.